### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by Ananyamadhu08**
- created navbar component 
- created footer component 
- created home-page

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,41 +1,29 @@
...
-            target="_blank"
-            rel="noreferrer"
-          >
-            Contribute
-          </a>
-        </div>
-      </header>
+      <Header />
+
+      <Routes>
```
Comment (VedanthB) [inline src/App.js:17]: you can move the routes into a diff file

File: src/App.js [hunk 2]
```diff
@@ -1,41 +1,29 @@
 import "./App.css";
-import logo from "./logo.png";
+import { Routes, Route } from "react-router-dom";
+import Home from "./routes/Home";
```
Comment (VedanthB) [inline src/App.js:3]: Suggestion : you can make barrel export for images where you export all the images in one file and import it from one file at here.

P.S: You can do barrel export for components ,pages, contexts etc..

File: src/components/Home-page/FeaturedBrands.jsx [hunk 3]
```diff
@@ -0,0 +1,55 @@
+import React from "react";
+
+function FeaturedBrands() {
+  return (
+    <div className="bg-rose-300 w-full p-16">
+      {/* <!-- featured brands --> */}
```
Comment (VedanthB) [inline src/components/Home-page/FeaturedBrands.jsx:6]: remove comments

**Review by VedanthB — APPROVED** [review]
Good work ananya, I have suggested a few changes please look into those.

**Review by iHarryD — APPROVED** [review]
- Code is extremely readable and clean.
- Use clamps and calc to remove media queries completely.
- Store repetitive transition declarations in variables.

**Pull request merged**